ts/torch_handler/base_handler.py

```diff
@@ -151,6 +155,12 @@ def initialize(self, context):
             self.map_location = "cpu"
             self.device = torch.device(self.map_location)
 
+        TORCH_EXPORT_AVAILABLE = False
```

Comment:
set these global variables in packaging if condition instead

Reply:
I looked at the logic again. This seems like the best place.